### DISCUSSION TITLE
Guacamole Labels No Longer Work

### DISCUSSION THREAD

**Discussion by sudo-kraken** [htpcBeginner/docker-traefik#145] [Support]
I have the below in use for the traefik labels for the guac service, I believe there to be an issue with the prefix being added as if I navigate to this locally I can reach and use Guacamole on IP:Port/guacamole. However it when accessing it at guad.domain.com it gives a white page and nothing else, the app logo appears in the browser tab above the url bar but nothing loads.

All other serivices are up and fucntional on traefik.

`      - "traefik.enable=true"
      ## HTTP Routers
      - "traefik.http.routers.guacamole-rtr.entrypoints=https"
      - "traefik.http.routers.guacamole-rtr.rule=Host(`guac.$DOMAINNAME`)"
      ## Middlewares
      - "traefik.http.routers.guacamole-rtr.middlewares=chain-authelia@file,add-guacamole"
      - "traefik.http.middlewares.add-guacamole.addPrefix.prefix=/guacamole"
      ## HTTP Services
      - "traefik.http.routers.guacamole-rtr.service=guacamole-svc"
      - "traefik.http.services.guacamole-svc.loadbalancer.server.port=8080"`

**Comment by SimpleHomelab** [comment]
Mine still works as of this morning. What traefik version?

Also what you describe is a common symptom of Guacamole not being able to connect to Mysql database. I would check that first.
